### DISCUSSION TITLE
245 as source of access point for manifestation

### DISCUSSION THREAD

**Discussion by GordonDunsire** [crystalyragui/MARC2RDA#376] [General]
This is a proposal for mapping the 245 tag to an access point for manifestation. It is essentially a string encoding scheme.

Only subfield strings from 245 are used. There is no attempt to disambiguate the resulting access point string, say as an authorised access point. The resulting string can be qualified and disambiguated ('authorised') using strings from other tags, particularly 250 (Edition statement) and 260 (Publication, distribution, etc.).

**Punctuation**

Remove all ISBD delimiter punctuation: the suffix punctuation for $a is too variable (", "; " /"; "; "; etc.)
Insert punctuation as indicated.

Normalise $a by removing non-filing characters using second indicator. Capitalise the first remaining letter.

Ignore strings and subfields following "=" anywhere in the tag; this is part of a separate (parallel) access point that cannot be parsed when the base parallel title is encoded in $b.

**Sequence of component strings**

$a, ($n, $p), $s, $f, $g, $k, $h

**String processing**

$a Title // normalised by removing non-filing characters.

"Proceedings ..."
"Interpretation of Zarahemla ..."
"Under the hill, or, The story of Venus and Tannhauser"
"Hamlet"
"The serpent--snapping eye"
"Diōnos Rōmaikōn historiōn eikositria biblia"
'Winter mind"' // cannot parse out the closing quote mark

$n + $p // prefix ". "

{
$n Number of part/section of a work

$p Name of part/section of a work // prefix ", "

"More!. 3"
"Faust. Part one"
"Bookman. Part B"
"Dissertation abstracts. A, The humanities and social sciences"
}

$p Name of part/section of a work (no $n) // prefix ". "

"Advanced calculus. Student handbook"
"Internationale Strassenkarte. Europe 1:2.5 Mio" // $b contains "=" and remainder of subfields are ignored
"Deutsche Bibliographie. Wöchentliches Verzeichnis. Reihe B, Beilage, Erscheinungen ausserhalb des Verlagsbuchhandels"
"Zentralblatt für Bakteriologie, Parasitenkunde, Infektionskrankheiten und Hygiene. 1. Abt. Originale. Reihe B, Hygiene, Krankenhaushygiene, Betriebshygiene, präventive Medizin"
"Annual report of the Minister of Supply and Service Canada under the Corporations and Labour Unions Returns Act. Part II, Labour unions"

$s Version // prefix ". "

"Director's report of the Association of Insurance Adjusters. Member release"

$f Inclusive dates // prefix ", "

"Short-Harrison-Symmes family papers, 1760-1878"
"Letter, 1901 March 6"

$g Bulk dates // enclose in parentheses

"Records, 1939-1973 (1965-1972)"
"Employment applications (Jan.-Dec. 1985)"

$k Form // enclose in brackets. There is no real distinction between medium and form as access point qualifiers, and the same punctuation will collocate similar access points.

"Four years at Yale, 1903 Sept. 16-1907 Oct. 5 [diaries]"
"Charity ball (1889) [typescript]"

$h Medium // enclose in brackets

"Green bag [microform]"
"New Lost City Ramblers with Cousin Emmy [sound recording]"
"Royal gazette [microform]"
"Geode [realia]"
"Printer's manual [microform]"
"Portals to the world. Selected Internet resources. Maldives$h[electronic resource]"
"PL 17 Hearing Files (1974). District 6 [Case Files] [microfilm (jacketted in fiche)]"

**Comment by CECSpecialistI** [comment]
This will result in many access points with the value "Report." Access point doesn't need to be unique like an authorized access point.

**Comment by CECSpecialistI** [comment]
Need string encoding schemes for access points for WEMI entities. PCC?

**Comment by AdamSchiff** [comment]
The only string encoding scheme that PCC will follow is for manuscripts.  And for them there will be an authorized access point constructed using the repository access point, followed by Manuscript and the manuscript item number.

British Library. ǂk Manuscript. ǂn Additional 10304
Bodleian Library. ǂk Manuscript. ǂn Bodley 34
Cambridge University Library. ǂk Manuscript. ǂn B.10.5
Magdalene College (University of Cambridge). ǂk Manuscript. ǂn Pepys 2125
University of Washington. ǂb Libraries. ǂk Manuscript. ǂn Ms. B

Manuscripts that are known by a title have an authorized access point for title:

Baldwin partbooks
variant access point: Christ Church (University of Oxford). ǂb Library. ǂk Manuscript. ǂn 979-983

Bruce codex
variant access point: Bodleian Library. ǂk Manuscript. ǂn Bruce 96
variant access point: Codex Brucianus
variant access point: Codex Bruce

Carmina Burana
variant access point: Bayerische Staatsbibliothek. ǂk Manuscript. ǂn Clm 4660
variant access point: Codex Latinus Monacensis 4660
variant access point: Codex Buranus

Codex Mexicanus
variant access point: Bibliothèque nationale de France. ǂk Manuscript. ǂn 23-24

Michigan medical codex
variant access point: University of Michigan. ǂb Library. ǂk Manuscript. ǂn P. Mich. 758
variant access point: University of Michigan. ǂb Library. ǂk Manuscript. ǂn P. Mich. inv. 21

Trinity carol roll
variant access point: Trinity College (University of Cambridge). ǂk Manuscript. ǂn O.3.58

Wickhambrook lute manuscript
variant access point: Yale University. ǂb Music Library. ǂk Manuscript. ǂn Music deposit 1


Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by GordonDunsire** [comment]
The purpose of this proposal is to preserve the context of $k and $h data with the other subfields in 245. It followed a discussion which pointed out the utility of the traditional "title proper" + "material designation" browse list (aka "intermediate title display").

The initial proposal says that the 'resulting string' (access point) can be qualified and disambiguated, but it is understood that qualification/disambiguation cannot be part of the transform because it would require an agreed system for authority control. No such agreement exists for manifestations or expressions. (No such agreement exists at global/RDA level for any entity; that is the failure of traditional top-down universal bibliographic control ;-)

Instead, we can provide a title browse access point that is latent in the MARC 21 subfield encodings. This represents a string encoding scheme with roots in ISBD and card catalogues.

Or we can leave it up to the post-transform application, as with all other entity access points and authority control. Without checking, I think a small amount of context would be lost ($k and $h collapse into 'category').

**Comment by CECSpecialistI** [comment]
Group discussion 2022-11-02: Proposal acceptable with addition of $b.

**Comment by tmqdeborah** [comment]
@Cypress asked me about including the $s in a 245 transformation for AP for manifestation. Since this element is based on a preferred title proper, it also raises the question of including the $s as part of a title proper. I did not include it for the AP for manifestation because the instructions for the subfield seem more appropriate for an edition qualifier than part of a title.

However, I just found this [DISCUSSION] (https://github.com/uwlib-cams/MARC2RDA/discussions/376). And then found this [DECISION ](https://github.com/uwlib-cams/MARC2RDA/wiki/2024-Meeting-Minutes#what-to-do-with-245-n-p-s) re. 245$s.

This discussion also [says to include the $b](https://github.com/uwlib-cams/MARC2RDA/discussions/376#discussioncomment-4038159)

I found only a few examples of 245$s in the LC file and the UW file that I have:
[245s_LC.txt](https://github.com/user-attachments/files/20884993/245s_LC.txt)
[245s_UW.txt](https://github.com/user-attachments/files/20884996/245s_UW.txt)

My current instructions for titles are:
Title245 (Title proper): anp
TitleWAP (130|240): adknp  + g if before flmors
TitleEAP (130|240): adknp  + g if before flmors + flmors + g (if after adknpt)

NameX00/Title:	$abcjqu + dg (if before $t) + $knpt + dg (if after $t and before 'flmors')
NameX10/Title:  $abcu + dgn (if before $t) + $kpt + dgn (if after $t and before 'flmors')
NameX11/Title:	$acequ + dgn (if before $t) + $kpt + dgn (if after $t and before 'flmors')

In a Uniform Title, $f and $s are not part of a title they are expression elements; so, do they really belong in a title proper?

Although $f (inclusive dates) can be part of a title for a single item, e.g., a letter or other unpublished manifestation/work
[245f_LC.txt](https://github.com/user-attachments/files/20894212/245f_LC.txt)
[245f_UW.txt](https://github.com/user-attachments/files/20894214/245f_UW.txt)

Can someone please tell me why the decision was made to include in the title proper:
- $s Version
- $b (perhaps only those preceded by = (parallel titles) or ; (subsequent titles)?

**Reply by lake44me** [reply]
Re $s, if the version information is on or in the manifestation and is just transcribed into this subfield because it is clearly version information, shouldn't it be included in Title Proper?  That would distinguish the title of this manifestation from the title of another manifestation for a different version....  

Re $b, it's iffy because usually it's the subtitle, but if it goes beyond that to description of the content, could be awkward.  RDA's definition of "subtitle" only pertains to text in alternate language to that spoken in a film, appearing at the bottom of the screen.  RDA's definition of "other title information" http://rdaregistry.info/Elements/m/P30142. probably aligns with MARC $b.  But it's not clear, in the definition of Title Proper, that a subtitle should be omitted.  This is an opportunity lost in my opinion - separating out subtitles from true "other title information" such as "with color illustrations by the author..."  For users, the subtitles are often important to distinguish many titles where the main part is a common word or phrase.

**Reply by GordonDunsire** [reply]
For an old discussion on some of this, see RDA-L: [Other title information and more in the official Toolkit](https://connect.ala.org/communities/community-home/digestviewer/viewthread?GroupId=113530&MessageKey=948a4a67-3302-4dc8-9dcc-90be217d6681&CommunityKey=1557c8d6-d568-4ba5-b72b-e9b99c7777f2&tab=digestviewer).

This thread is not about title proper, it is about access point for manifestation. They are not the same, although title proper is usually the basis of an authorized access point for manifestation.

**Reply by tmqdeborah** [reply]
@GordonDunsire, I posted my comment to this thread because if we include the $s and /or $f as a part of a title proper, then it will automatically be included as part of the access point for manifestation. So, the questions are:
- should information that is recorded in a $s or $f be included in a title proper 
- if not, should that information be included as a qualifier in an access point for manifestation?

@lake44me, I agree that there are many examples of data being recorded in a $b as other title information when (I think) it would be much more useful to record it as part of a title proper, e.g.
245 10 $a Thomas A. Edison : $b the world's greatest inventor
245 10 $a Thomas A. Edison, the world's greatest inventor

Maybe we could consider adding variant titles and APs for them, in Phase II? 

And I would definitely like to see parallel and subsequent titles included in variant titles and APs in Phase II.

**Reply by GordonDunsire** [reply]
@tmqdeborah: I was only referring to the context of subfield $b. There is no programmatic method that I can think of that will distinguish the content of the subfield as a 'subtitle' (and perhaps part of a title proper), as an ISBD dumping ground for 'information that may be asssociated with a title statement' (such as scale information), or the string that appears after the title statement and before the statement of responsibility. I am using legacy terminology of course. We discussed this earlier in the project, as well as on other forums.

**Comment by CECSpecialistI** [comment]
Meeting 2025-07-02 DECISION: Ignore 245 $f and $s for titles proper for non-collection works. 

For access points when there is no 130 or 240: Include $f as a qualifier if record is for a collection work. Include $s as a qualifier.

UW: do an indication rule on 245 $f or 245 $s and do a cleanup of these records. 

This should go into pre-transform cleanup for libraries. Flag as a cleanup. Then maybe reach out to source institutions to ask for a reason why this is being done.

**Reply by tmqdeborah** [reply]
> Meeting 2025-07-02 DECISION: Ignore 245 $f and $s for titles proper for non-collection works.
> 
> For access points when there is no 130 or 240: Include $f as a qualifier if a record describes a collection work. Include $s as a qualifier.

Rephrasing and expanding:
  - For a collection work expression, and manifestation:
    - title: 
      - do not include $b 
      - include $f and $s
    - access point:
      - do not include $b 
      - include $f and $s
  - for a non-collection manifestation:
      - title: 
      - do not include $b, $f, or $s
    - access point:
      - do not include $b or $f
      - include $s

But, if $s contains version information and we are going to map it as a qualifier for an AP for manifestation, then should we also map 250$a version information as a qualifier for an AP for manifestation?

And, if so, should we map both 245$s and 250$a as qualifiers for Expression APs? And for Aggregating Work APs?

And should we map 245$s as 'designation of version'?

**Comment by CECSpecialistI** [comment]
We'll revisit $b in Phase II.
